Makefile

```diff
@@ -0,0 +1,9 @@
+exec: *.go */*.go go.*
+	# the executable
+	go build -o $@ -ldflags "-s -w"
+	file $@
```

Comment:
What's this line for? `man file` says it's for printing out what type of file this likely is, but I don't see why that would be needed during compilation.